lib/dal/src/prop.rs

```diff
@@ -375,7 +378,7 @@ impl Prop {
                     _ => return Err(PropError::PropParentInvalid(prop_id)),
                 },
             ),
-            None => Ok(None),
+            None => Err(PropError::PropIsOrphan(prop_id)),
```

Comment:
Found this along the way. Could've been painful in the future, now there's an error.